.github/workflows/twister_tests_blackbox.yml

```
@@ -51,7 +51,7 @@ jobs:
         toolchains: all
 
     - name: Run Pytest For Twister Black Box Tests
-      if: ${{ startsWith(runner.os, 'ubuntu') }}
+      if: ${{ startsWith(matrix.os, 'ubuntu') }}
```

should this be

```suggestion
      if: ${{ runner.os == "Linux" }}
```
?

Other checks use that pattern and right now matrix.os is only used for `runs-on`